packages/overlay/src/integrations/sentry/components/performance/PerformanceTabDetails.tsx

```diff
@@ -58,7 +58,8 @@ export default function PerformanceTabDetails() {
           <Route path="webvitals" element={<WebVitals />} />
           <Route path="webvitals/:page" element={<WebVitalsDetail />} />
           {/* Default tab */}
-          <Route path="*" element={<Queries showAll={showAll} />} />
+          <Route path="queries" element={<Queries showAll={showAll} />} />
+          <Route path="*" element={<Navigate to="/performance/queries" replace />} />
```

Comment:
Kind of weird that we need to do this.

But thanks for the fix, it's been annoying me 😅